docs/index.rst

```diff
@@ -1,62 +1,34 @@
 
-Welcome to Rikai's documentation!
-=================================
-
-Rikai is an `Apache Spark`_ based ML data format built for working with
-unstructured data at scale. Processing large amounts of data for ML is never trivial, but that
-is especially true for images and videos often at the core of deep learning applications. We are
-building Rikai with two main goals:
-
-1. Enable ML engineers/researchers to have a seamless workflow from feature engineering (`Spark`_)
-   to training (`PyTorch`_/`Tensorflow`_), from notebook to production (:doc:`See details <types>`).
-2. Enable advanced analytics capabilities to support much faster active learning, model debugging,
-   and monitoring in production.
-
-.. code-block:: python
-
-  from pyspark.ml.linalg import DenseMetrix
-  from rikai.types import Image, Box2d
-  from rikai import numpy as np
-
-  df = spark.createDataFrame(
-      [{
-          "id": 1,
-          "mat": DenseMatrix(2, 2, range(4)),
-          "image": Image("s3://foo/bar/1.png"),
-          "annotations": [
-              {
-                  "label": "cat",
-                  "mask": np.random(size=(256,256)),
-                  "bbox": Box2d(xmin=1.0, ymin=2.0, xmax=3.0, ymax=4.0)
-              }
-          ]
-      }]
-  )
-
-  df.write.format("rikai").save("dataset/out")
-
-``Rikai`` dataset can be seamlessly integrated into your favorite training frameworks,
-taking `Pytorch`_ as an example:
-
-.. code-block:: python
-
-  from rikai.torch import DataLoader
-
-  data_loader = DataLoader(
-      "dataset/out",
-      shuffle=True,
-      batch=8,
-  )
-  for examples in data_loader:
-      print(example)
-
-Additionally, the parquet-native nature of the ``rikai`` format allows such unstructured ML dataset
-being analyzed in `Jupyter`_, `Spark`_, `Presto`_ or
-`BigQuery <https://cloud.google.com/bigquery/external-data-cloud-storage>`_.
+Welcome to Rikai!
+=================
+
+What is Rikai
+-------------
+
+The success of a AI product is determined by high quality of ML data.
+This is the number one reason ML practioners spending the most of the efforts working on data instead
+of innovating on models.
+Rikai alleviates ML practioners from many of these tedious data tasks so that they can
+focus on model innovation and problem solving.
+
+Rikai is a large-scale data system specifically designed for Machine Learning workflows,
+specialized in Deep Learning development over various forms of unstructured data,
+for example, image, video or sensor data.
+
+Rikai strives to offer great developer experience to assist ML engineers at each
```

Comment:
## Format

The core of Rikai is a data format ("rikai format") based on [Apache Parquet](https://parquet.apache.org/). Rikai augments parquet with a rich collection of semantic types design specifically for unstructured data and annotations. 

## Data Integrations

Rikai comes with an extensive set of I/O connectors. For ETL, Rikai is able to consume popular formats like ROS bags and Coco. For analysis, it's easy to read Rikai data into pandas/spark DataFrames (Rikai handles serde for the semantic types). And for training, Rikai allows direct creation of Pytorch/Tensorflow datasets without manual conversion.

## SQL-ML Engine

Rikai extends Spark SQL with ML capability which allows users to analyze Rikai datasets using own models with SQL ("Bring your own model")

## Visualization

Carefully crafted data-visualization embedded with semantic types, especially in Jupyter notebooks, to help you visualize and inspect your AI data without having to remember complicated raw image manipulations.

Reply:
Good point. Updated.